mautrix_telegram/abstract_user.py

```diff
@@ -201,6 +201,7 @@ async def _update_catch(self, update: TypeUpdate) -> None:
             self.log.exception("Failed to handle Telegram update")
             UPDATE_ERRORS.labels(update_type=update_type).inc()
         UPDATE_TIME.labels(update_type=update_type).observe(time.time() - start_time)
+        self.brige.confirm_bridge_liveness()
```

Comment:
This is the only place that needs to call the confirm?

Also, typo:

```suggestion
        self.bridge.confirm_bridge_liveness()
```